lab_1_classify_by_unigrams/main.py

```diff
@@ -6,49 +6,94 @@
 
 def tokenize(text: str) -> list[str] | None:
     """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+        Splits a text into tokens, converts the tokens into lowercase,
```

Comment:
revert changes in all doc strings, please

Reply:
!

Reply:
reverted the excessive tabulation